src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -89,6 +91,24 @@ public function panelAction(Request $request, $token)
             return new Response($this->twig->render('@WebProfiler/Profiler/info.html.twig', ['about' => 'no_token', 'token' => $token, 'request' => $request]), 200, ['Content-Type' => 'text/html']);
         }
 
+        if ($profile->hasCollector($bestPanel = '_best')) {
+            @trigger_error(sprintf('The "%s" data collector name is reserved for internal usage since Symfony 4.4. Please rename it.', $bestPanel));
```

Comment:
This is triggering a warning, not a deprecation.

And this should rather be detected at compile time when registering collectors, to warn early rather than when someone tries to load that link.

Reply:
reserving a name only when the UI displaying profiles is used is even worse IMO. I don't know of any project wanting to use the profiler system without ever feeding profiles to the UI displaying them.

Reply:
but anyway, see my comment suggesting an alternative which does not need reserving a name.